src/components/CubeDraft.js

```diff
@@ -124,22 +124,33 @@ const CubeDraft = ({ draft, socket }) => {
       setLoading(false);
 
       if (seat === '0') {
-        let status = 'in_progress';
-        setInterval(async () => {
-          try {
-            if (status === 'in_progress') {
+        // the trybotpicks response will eventually flip this to 'done'
+        let status = 'inProgress';
+        // let's ensure we only make one trybotpicks call at a time
+        let keepRunning = true
+        while (keepRunning) {
+            keepRunning = false
+            if (status === 'inProgress') {
+              try {
+              setInterval(async () => { 
```

Comment:
this looks like it's going to keep setting up loops and making the problem worse